flowz/app.py

```diff
@@ -89,12 +90,11 @@ def run(self):
         self.loop.spawn_callback(self.main)
         self.loop.start()
         if self.exc_info:
-            raise self.exc_info[1], None, self.exc_info[2]
```

Comment:
By the way, my attempt to fix this led me to the conclusion that the original line seemed to have a bug. The first argument to the `raise` probably should have been `self.exc_info[1]`.